containers/Ideas/ExplorePage.js

```diff
@@ -41,17 +54,24 @@ export const ExplorePage = (props) => {
     return projectsListener
   }, [])
 
+  useEffect(() => {
+    fetchAuthenticatedUser()
+  }, [])
+
   const toProfile = () => {
-    props.navigation.navigate('ProfileScreen', {name: 'Bob'})
+    props.navigation.navigate('ProfileScreen', {userId: user.pk})
   }
 
   return (
     <View style={styles.container}>
-      <ButtonAvatar
-        labelText="profile"
-        hintText="click to go to profile and settings"
-        onPress={toProfile}
-      ></ButtonAvatar>
+      {user &&
+        <ButtonAvatar
+          imgSource={{ uri: user._avatar }}
```

Comment:
so that works for local and dev now?
at least in iOS it shows the avatar connected to local api.

Reply:
Yes, the user API currently gives the whole URL, so it does work. 
I only checked the comments to find out about the fallback and was confused, because that API does not give the whole URL, but only the bit after the base. I will dive into that deeper for the fallback, though.